src/components/override-sdk/infra/Assignment/Assignment.tsx

```diff
@@ -172,6 +178,17 @@ export default function Assignment(props) {
     setErrorMessages([...errorStateProps]);
   }
 
+  function clearErrors() {
+    errorMessages.forEach(error =>
+      PCore.getMessageManager().clearMessages({
+        property: error.message.clearMessageProperty,
+        pageReference: error.message.pageRef,
+        context: containerID,
+        type: 'error'
+      })
```

Comment:
I can see from documentation "category" is also required field.
https://docs.pega.com/bundle/pcore-pconnect-88/page/pcore-pconnect-public-apis/api/messagesconfigobject.html

Reply:
Works without it for some reason (you can also see it being utilised on the FieldGroupTemplate.tsx line 25). However I've added the category as 'Property' to adhere to the documentation.